phases/04-computer-vision/20-image-retrieval-metric/outputs/prompt-retrieval-loss-picker.md

```
@@ -0,0 +1,49 @@
+## Output
+
+```
+[loss]
+  name:       triplet | InfoNCE | ProxyNCA | ProxyAnchor
+  margin:     <float, if triplet>
+  temperature: <float, if InfoNCE>
+  embedding_dim: typical 128-768
+
+[training]
+  batch:      <int>
+  optimiser:  Adam / SGD with weight decay
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)